doc/classes/RandomNumberGenerator.xml

```diff
@@ -28,6 +28,17 @@
 				Generates pseudo-random float between [code]from[/code] and [code]to[/code].
 			</description>
 		</method>
+		<method name="randfn">
+			<return type="float">
+			</return>
+			<argument index="0" name="deviation" type="float" default="1.0">
+			</argument>
+			<argument index="1" name="mean" type="float" default="0.0">
+			</argument>
+			<description>
+				Generates normally distributed pseudo-random number with a standart [code]deviation[/code] and the specified [code]mean[/code].
```

Comment:
`standart` -> `standard`

Reply:
Also, probably mention that the Box-Muller transform is being used.